api/allIncidents/adminIncidentsRouter.js

```diff
@@ -173,8 +173,11 @@ router.post(
   validateAndSanitizeIncidentObject,
   async (req, res, next) => {
     Incidents.createIncident(req.sanitizedIncident)
-      .then(() => {
-        res.status(201).json({ message: 'Incident Successfully Created' });
+      .then((result) => {
+        res.status(201).json({
```

Comment:
Let's go ahead and review this one with Paul.

Reply:
I think this is a simple and necessary change that will dramatically improve the user experience.